hw12_13_14_15_calendar/internal/server/grpc/server.go

```diff
@@ -61,13 +61,14 @@ func (s *Server) Start() error {
 		return err
 	}
 
-	s.logg.Info("HTTP server run %s:%s", s.host, s.port)
+	s.logg.Info("GRPS server run %s:%s", s.host, s.port)
```

Comment:
```suggestion
	s.logg.Info("GRPС server run %s:%s", s.host, s.port)
```
Remote Procedure Call